.github/workflows/release.yml

```diff
@@ -0,0 +1,128 @@
+name: release
```

Comment:
We aren’t building on tags, we’re building based off of the next version as determined by semantic release. This setup is needed to allow for attaching release artifacts to the release created by semantic release and preventing a release from getting created if the container image builds failed. Otherwise, semantic release creates tags and a release (without the artifacts), and only then the containers will get built. If that build then fails you have a release with no associated images.